[2025-01-13T14:01:56.438+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: finflow_source_data_pipeline.create_dataset manual__2025-01-13T14:01:15.378177+00:00 [queued]>
[2025-01-13T14:01:56.452+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: finflow_source_data_pipeline.create_dataset manual__2025-01-13T14:01:15.378177+00:00 [queued]>
[2025-01-13T14:01:56.452+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-01-13T14:01:56.470+0000] {taskinstance.py:1382} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2025-01-13 14:01:15.378177+00:00
[2025-01-13T14:01:56.480+0000] {standard_task_runner.py:57} INFO - Started process 1418 to run task
[2025-01-13T14:01:56.487+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'finflow_source_data_pipeline', 'create_dataset', 'manual__2025-01-13T14:01:15.378177+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/finflow.py', '--cfg-path', '/tmp/tmp4a2_2osh']
[2025-01-13T14:01:56.491+0000] {standard_task_runner.py:85} INFO - Job 158: Subtask create_dataset
[2025-01-13T14:01:56.577+0000] {task_command.py:416} INFO - Running <TaskInstance: finflow_source_data_pipeline.create_dataset manual__2025-01-13T14:01:15.378177+00:00 [running]> on host ab4e86dc450e
[2025-01-13T14:01:56.748+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='finflow_source_data_pipeline' AIRFLOW_CTX_TASK_ID='create_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-01-13T14:01:15.378177+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-13T14:01:15.378177+00:00'
[2025-01-13T14:01:56.752+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-01-13T14:01:56.753+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-01-13T14:01:56.755+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-01-13T14:01:56.760+0000] {bigquery.py:468} INFO - datasetId was not specified in `dataset_reference`. Will use default value finflow_raw.
[2025-01-13T14:01:56.760+0000] {bigquery.py:468} INFO - projectId was not specified in `dataset_reference`. Will use default value finflow-analytics-production.
[2025-01-13T14:01:56.761+0000] {bigquery.py:479} INFO - Creating dataset: finflow_raw in project: finflow-analytics-production 
[2025-01-13T14:01:57.420+0000] {bigquery.py:483} INFO - Dataset created successfully.
[2025-01-13T14:01:57.455+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=finflow_source_data_pipeline, task_id=create_dataset, execution_date=20250113T140115, start_date=20250113T140156, end_date=20250113T140157
[2025-01-13T14:01:57.505+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-13T14:01:57.547+0000] {taskinstance.py:2778} INFO - 6 downstream tasks scheduled from follow-on schedule check
